### ISSUE TITLE
Accept stream input

### ISSUE THREAD

**Issue by zlangner** [GowenGit/docnet#58]
Currently it appears that this library supports byte[] and filepath inputs. It would be much more flexible if it also allowed for Stream input.

**Comment by Modest-as** [comment]
Hi, PDFium does not expose a way to pass streams to the unmanaged code. Similar issue : #45

**Comment by zlangner** [comment]
That is true but that doesn't mean the API layer of docnet can't. You could do [what PDFiumSharp does](https://github.com/ArgusMagnus/PDFiumSharp/blob/master/PDFiumSharp/Types/FPDF_FILEACCESS.cs) and offer methods that take streams but resolve them into byte arrays before accessing the unmanaged code.
